ledger/benches/blockstore_processor.rs

```diff
@@ -162,6 +162,7 @@ fn bench_execute_batch(
                 &mut timing,
                 None,
                 &prioritization_fee_cache,
+                None::<fn() -> PreCommitCallbackResult<Option<usize>>>,
```

Comment:
Unrelated aside - I always (perhaps naively) assumed that `fn()` were just "raw" function **pointers**. but they are actually assumed to be valid (similar to `NonNull` pointers), and so `Option<fn() -> ()>` will actually still only be 8-bytes